### ISSUE TITLE
Trouble running unit tests in Windows

### ISSUE THREAD

**Issue by mcandre** [hashicorp/packer#6461]
When I try to run the unit test suite in Windows, I get an error about long filenames.

```console
C:\>go version
go version go1.8.3 windows/amd64

C:\>go test
go build github.com/hashicorp/packer/vendor/github.com/oracle/oci-go-sdk/core: C:\Go\pkg\tool\windows_amd64\compile.exe: fork/exec C:\Go\pkg\tool\windows_amd64\compile.exe: The filename or extension is too long.
```

**Comment by DanHam** [comment]
@mcandre Try updating to the latest version of Go.

I suspect you are encountering the issue reported in #6220

**Comment by arizvisa** [comment]
Definite dupe of #6220

**Comment by SwampDragons** [comment]
Fixed by golang 1.11 which was released last week.
